providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/e2e/RunFlagdInProcessSSLCucumberTest.java

```diff
@@ -0,0 +1,27 @@
+package dev.openfeature.contrib.providers.flagd.e2e;
+
+import org.apache.logging.log4j.core.config.Order;
+import org.junit.jupiter.api.Disabled;
+import org.junit.platform.suite.api.ConfigurationParameter;
+import org.junit.platform.suite.api.IncludeEngines;
+import org.junit.platform.suite.api.SelectClasspathResource;
+import org.junit.platform.suite.api.Suite;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import static io.cucumber.junit.platform.engine.Constants.GLUE_PROPERTY_NAME;
+import static io.cucumber.junit.platform.engine.Constants.PLUGIN_PROPERTY_NAME;
+
+/**
+ * Class for running the reconnection tests for the RPC provider
+ */
+@Order(value = Integer.MAX_VALUE)
+@Suite(failIfNoTests = false)
+@IncludeEngines("cucumber")
+//@SelectClasspathResource("features/evaluation.feature")
```

Comment:
i do not get this working, maybe i am wrong here, but if somebody has time to take a look this would be highly appreciated

Reply:
seems like, flagd currently does not support SSL for the sync server